lua/neotest/consumers/run.lua

```diff
@@ -197,6 +197,16 @@ function neotest.run.adapters()
   return client:get_adapters()
 end
 
+--- Get last test run tree and adapter id.
+---@return neotest.Tree | nil, integer | nil
+function neotest.run.get_last_run()
```

Comment:
@rcarriga: still seems a bit strange to have a public `.run()` that returns a `tree, adapter_id`? But have to admit I didn't spend enough time to really understand the code's data structures and API's properly..

Reply:
Sorry I should have been clearer :facepalm: The function just return `position_id, last_args`, not the tree as yes you're right the tree is not really meant for general usage, just for consumers.